vnext/Shared/OInstance.cpp

```diff
@@ -541,18 +543,21 @@ InstanceImpl::~InstanceImpl() {
 std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules(
     std::shared_ptr<MessageQueueThread> nativeQueue) {
   std::vector<std::unique_ptr<NativeModule>> modules;
+  auto transitionalProps{ReactPropertyBagHelper::CreatePropertyBag()};
 
   modules.push_back(std::make_unique<CxxNativeModule>(
       m_innerInstance,
       Microsoft::React::GetHttpModuleName(),
-      [nativeQueue]() -> std::unique_ptr<xplat::module::CxxModule> { return Microsoft::React::CreateHttpModule(); },
+      [nativeQueue, transitionalProps]() -> std::unique_ptr<xplat::module::CxxModule> {
```

Comment:
>transitionalProps [](http://example.com/codeflow?start=20&length=17)

Do we want to share the property bag across modules ? Would this lambda capture end up copying the property bag ?

Reply:
Yes, and yes.

On the deep dive describing this architecture, we decided to use an exclusive property bag to share objects between CxxModules.

Reply:
Ah .. it looks the copy is shallow by default .. makes sense ..